Common subdirectories: a/bitshuffle-0.3.5 and b/bitshuffle-0.3.5
Common subdirectories: a/imagecodecs and b/imagecodecs
Common subdirectories: a/imagecodecs.egg-info and b/imagecodecs.egg-info
Common subdirectories: a/liblzf-3.6 and b/liblzf-3.6
diff -u a/setup.py b/setup.py
--- a/setup.py	2020-02-03 19:56:43.000000000 -0500
+++ b/setup.py	2020-05-27 21:35:52.155134630 -0400
@@ -129,108 +129,16 @@
     #     ),
 }
 
-if os.environ.get('COMPUTERNAME', '').startswith('CG-'):
-    # Windows development environment. Use static libraries
-    from _inclib import INCLIB
-    INCLUDE_DIRS.append(INCLIB)
-    LIBRARY_DIRS.append(INCLIB)
-
-    EXTENSIONS['aec'] = dict(libraries=['libaec'])
-    EXTENSIONS['bz2'] = dict(libraries=['libbz2'])
-    EXTENSIONS['lzf'] = dict(libraries=['lzf'])
-    EXTENSIONS['gif'] = dict(libraries=['libgif'])
-    # EXTENSIONS['szip'] = dict(libraries=['libaec'])
-    EXTENSIONS['zstd'] = dict(libraries=['zstd_static'])
-    EXTENSIONS['jpegls']['define_macros'] = [('CHARLS_STATIC', 1)]
-    EXTENSIONS['jpeg2k']['define_macros'] += [('OPJ_STATIC', 1)]
-    EXTENSIONS['jpeg2k']['include_dirs'] = [INCLIB + 'openjpeg-2.3']
-    EXTENSIONS['jpegxr']['include_dirs'] = [INCLIB + 'jxrlib']
-    EXTENSIONS['blosc'] = dict(
-        libraries=['libblosc', 'zlib', 'lz4', 'snappy', 'zstd_static']
-    )
-    EXTENSIONS['brotli'] = dict(
-        libraries=[
-            'brotlienc-static', 'brotlidec-static', 'brotlicommon-static'
-        ]
-    )
-    EXTENSIONS['lzma'] = dict(
-        libraries=['lzma-static'],
-        define_macros=[('LZMA_API_STATIC', 1)]
-    )
-    EXTENSIONS['tiff'] = dict(
-        libraries=[
-            'tiff', 'z', 'jpeg', 'png', 'webp', 'zstd_static', 'lzma-static',
-        ],
-        define_macros=[('LZMA_API_STATIC', 1)]
-    )
-    EXTENSIONS['jpegxl'] = dict(
-        libraries=[
-            'brunslidec-c', 'brunslienc-c',
-            # static linking
-            'brunslidec-static', 'brunslienc-static', 'brunslicommon-static',
-            # vendored brotli currently used for compressing metadata
-            'brunsli_brotlidec-static',
-            'brunsli_brotlienc-static',
-            'brunsli_brotlicommon-static',
-        ]
-    )
-
-elif os.environ.get('LD_LIBRARY_PATH', os.environ.get('LIBRARY_PATH', '')):
-    # Czaki's CI environment
-    del EXTENSIONS['jpeg12']
-
-    base_path = os.environ.get(
-        'BASE_PATH', os.path.dirname(os.path.abspath(__file__))
-    )
-    include_base_path = os.path.join(
-        base_path, 'build_utils/libs_build/include'
-    )
-    LIBRARY_DIRS = [
-        x for x in os.environ.get(
-            'LD_LIBRARY_PATH', os.environ.get('LIBRARY_PATH', '')
-        ).split(':') if x
-    ]
-
-    if os.path.exists(include_base_path):
-        INCLUDE_DIRS.append(include_base_path)
-        for el in os.listdir(include_base_path):
-            path_to_dir = os.path.join(include_base_path, el)
-            if os.path.isdir(path_to_dir):
-                INCLUDE_DIRS.append(path_to_dir)
-        jxr_path = os.path.join(include_base_path, 'libjxr')
-        if os.path.exists(jxr_path):
-            jpegxr_include_dirs = [jxr_path]
-            for el in os.listdir(jxr_path):
-                path_to_dir = os.path.join(jxr_path, el)
-                if os.path.isdir(path_to_dir):
-                    jpegxr_include_dirs.append(path_to_dir)
-            EXTENSIONS['jpegxr']['include_dirs'] = jpegxr_include_dirs
-
-    for dir_path in INCLUDE_DIRS:
-        if os.path.exists(os.path.join(dir_path, 'charls', 'charls.h')):
-            break
-    else:
-        del EXTENSIONS['jpegls']
-
-    for dir_path in INCLUDE_DIRS:
-        if os.path.exists(os.path.join(dir_path, 'zfp.h')):
-            break
-    else:
-        del EXTENSIONS['zfp']
-
-    EXTENSIONS['jpegxr']['define_macros'] = [('__ANSI__', 1)]
-
-else:
-    # Most recent Debian
-    del EXTENSIONS['jpeg12']  # jpeg12 library not available
-    del EXTENSIONS['jpegls']  # CharLS 2.1 library not available
-    del EXTENSIONS['jpegxl']  # Brunsli library not available
-    del EXTENSIONS['zfp']  # ZFP library not available
-
-    if sys.platform != 'win32':
-        EXTENSIONS['jpeg2k']['include_dirs'] = ['/usr/include/openjpeg-2.3']
-        EXTENSIONS['jpegxr']['include_dirs'] = ['/usr/include/jxrlib']
-        EXTENSIONS['jpegxr']['define_macros'] = [('__ANSI__', 1)]
+# Most recent Debian
+# del EXTENSIONS['jpeg12']  # jpeg12 library not available
+# del EXTENSIONS['jpegls']  # CharLS 2.1 library not available
+# del EXTENSIONS['jpegxl']  # Brunsli library not available
+# del EXTENSIONS['zfp']  # ZFP library not available
+
+# if sys.platform != 'win32':
+#     EXTENSIONS['jpeg2k']['include_dirs'] = ['/usr/include/openjpeg-2.3']
+#     EXTENSIONS['jpegxr']['include_dirs'] = ['/usr/include/jxrlib']
+#     EXTENSIONS['jpegxr']['define_macros'] = [('__ANSI__', 1)]
 
 ###############################################################################
 
Common subdirectories: a/tests and b/tests
